"""v8 added the supplier table to check if the tuple error was removed

Revision ID: 7171446f8acd
Revises: 11ddc8d5afd0
Create Date: 2020-07-20 08:02:33.267376

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = '7171446f8acd'
down_revision = '11ddc8d5afd0'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.create_table('supplier_tb',
    sa.Column('ID', sa.Integer(), autoincrement=True, nullable=False),
    sa.Column('SupplierName', sa.String(length=50), nullable=False),
    sa.Column('Email', sa.String(length=50), nullable=False),
    sa.Column('TelephoneNo', sa.String(length=50), nullable=False),
    sa.Column('Location', sa.String(length=50), nullable=False),
    sa.PrimaryKeyConstraint('ID')
    )
    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_table('supplier_tb')
    # ### end Alembic commands ###
